src/replication.c

```diff
@@ -685,10 +648,6 @@ replica_receiver(void *arg) {
 			}
 			MTX_UNLOCK(&spec->rq_mtx);
 		}
-		//Check if number of available replicas have changed
```

Comment:
replica cleanup need to happen in case of any errors while reading in this thread as part of cstor#143 bug.. we don't need this as part of this PR